src/hooks/useQueryMutationClient.ts

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { IDeskproClient, useDeskproAppClient } from "@deskpro/app-sdk";
+import { useMutation, UseMutationOptions } from "@tanstack/react-query";
+
+export const useQueryMutationWithClient = <
```

Comment:
use `useQueryMutationWithClient` from `app-sdk`

Reply:
not exported

Reply:
But you added this hook into `app-sdk`. Then please fix it

Reply:
Not sure why it was renamed to useMutation

Reply:
please, check it in the `app-sdk`, fix it if there is a bug and import it here